libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -303,6 +307,7 @@ private static bool ChannelSupportsOAuthCard(string channelId)
                 case Channels.Cortana:
                 case Channels.Skype:
                 case Channels.Skypeforbusiness:
+                case Channels.Msteams:
                     return false;

```

Comment:
AFAIK Teams supports OAuthCard!?!?

Reply:
@tomlm I believe the SSO flow uses the OAuth Prompt, rather than the OAuth card. I know Carlos / Eric / Swagat have gone pretty deep here, and this is where they settled. @EricDahlvang  Can you elaborate a bit?

Reply:
Tom is correct... this was in unintentional change.  (Teams does now support OAuthCards, as long as the SignInLink is present)

NOTE: after seeing Tom's comment here, i've also removed Channels.Msteams from **OAuthInput**